docs/code/Flakes/Tutorials/Hacking.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 03-Create a Development Environment
+date: 2021-02-09
+---
+
+## Hacking
+
+`nix-shell` has been replaced by `nix develop`, and by default it builds the attribute `devShell.x86_64-linux`, so a simple python development environment would look like this:
```

Comment:
Could we add a link to the [`nix develop` man page](https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-develop.html) ?

Reply:
Addressed in e6bbfe1.